docs/docs/10.10-shallow-compare.md

```diff
@@ -25,8 +25,8 @@ export class SampleComponent extends React.Component {
 }
 ```
 
-`shallowCompare` performs a shallow equality check on the current `props` and `nextProps` objects as well as the current `state` and `nextState` objects.  
+`shallowCompare` performs a shallow equality check on the current `props` and `nextProps` objects as well as the current `state` and `nextState` objects.
```

Comment:
Please revert this and the other change here. It looks intentional, though it's still not great formatting when rendered (I'd rather leave it and fix separately). I'm guessing this is just a result of saving in your editor. I encourage you to install a plugin that supports the editorconfig file we have which overrides that built-in behavior (http://editorconfig.org/#download)